ui/raidboss/data/05-shb/raid/e10s.js

```diff
@@ -201,15 +233,56 @@ export default {
       netRegexDe: NetRegexes.startsUsing({ source: 'Schattenkönig', id: '5B2C', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ source: 'Roi De L\'Ombre', id: '5B2C', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ source: '影の王', id: '5B2C', capture: false }),
-      alertText: (data, _, output) => output.text(),
+      durationSeconds: (data) => data.gigaSlashCleaveDebuffDuration,
+      alertText: (data, _, output) => {
+        let ret = output.default;
+        switch (data.gigaSlashCleaveDebuffId) {
+        case '973':
+          ret = output.westCleave;
+          break;
+        case '974':
+          ret = output.eastCleave;
+          break;
+        case '975':
+          ret = output.northCleave;
+          break;
+        case '976':
+          ret = output.southCleave;
+          break;
+        }
+        delete data.gigaSlashCleaveDebuffId;
+        delete data.gigaSlashCleaveDebuffDuration;
+        return ret();
+      },
       outputStrings: {
-        text: {
+        northCleave: {
+          en: 'North Cleave',
+        },
+        southCleave: {
+          en: 'South Cleave',
+        },
+        eastCleave: {
+          en: 'East Cleave',
+        },
+        westCleave: {
+          en: 'West Cleave',
+        },
+        default: {
           en: 'Right Cleave',
           de: 'Rechter Cleave',
           ko: '오른쪽 장판',
         },
       },
     },
+    {
+      id: 'E10S Shadow Servant Cleave Drop',
+      netRegex: NetRegexes.gainsEffect({ effectId: '97[3456]' }),
+      condition: Conditions.targetIsYou(),
+      run: (data, _) => {
+        data.gigaSlashCleaveDebuffId = _.effectId;
+        data.gigaSlashCleaveDebuffDuration = _.duration;
```

Comment:
This should probably be `(data, matches)`, since we're actually using the parameter and not just skipping over it to `output`.

Reply:
Renamed variable.